tests/plugins/test_koji_import.py

```diff
@@ -1841,3 +1842,55 @@ def test_koji_import_primary_images(self, tmpdir, os_env, add_tag_conf_primaries
             return
 
         runner.run()
+
+    @pytest.mark.parametrize(('comp', 'sign_int', 'override'), [
+        ([1, 2, 3], "beta", True),
+        ([2, 3, 4], "release", True),
+        ([3, 4, 5], "beta", False),
+        ([4, 5, 6], "release", False),
+        (None, None, None)
+    ])
+    def test_odcs_metadata_koji(self, tmpdir, os_env, comp, sign_int, override):
+        session = MockedClientSession('')
+        tasker, workflow = mock_environment(tmpdir,
+                                            name='ns/name',
+                                            version='1.0',
+                                            release='1',
+                                            session=session)
+
+        resolve_comp_entry = False
+        if comp is not None and sign_int is not None and override is not None:
+            resolve_comp_entry = True
+
+            workflow.prebuild_results[PLUGIN_RESOLVE_COMPOSES_KEY] = {
+                'composes': comp,
+                'signing_intent': sign_int,
+                'signing_intent_overridden': override,
+            }
+        else:
+            workflow.prebuild_results[PLUGIN_RESOLVE_COMPOSES_KEY] = None
```

Comment:
Tests shouldn't set results to None in this case - as plugin might not run at all

Reply:
ahh right, I can remove that

Reply:
We should test both cases.

Reply:
ok